pages/home.tsx

```diff
@@ -1,115 +1,5 @@
-import { useEffect } from 'react';
-import { graphql, useLazyLoadQuery } from 'react-relay';
-import { fetchQuery } from 'relay-runtime';
-
-import { ITEMS_PER_PAGE, MAX_PIECES_DISPLAYED_PER_FEED_EVENT } from '~/components/Feed/constants';
-import { FeedMode } from '~/components/Feed/Feed';
-import { NOTES_PER_PAGE } from '~/components/Feed/Socialize/NotesModal/NotesModal';
-import { FEED_MODE_KEY } from '~/constants/storageKeys';
-import { FeedNavbar } from '~/contexts/globalLayout/GlobalNavbar/FeedNavbar/FeedNavbar';
-import { homeQuery } from '~/generated/homeQuery.graphql';
-import usePersistedState from '~/hooks/usePersistedState';
-import GalleryRoute from '~/scenes/_Router/GalleryRoute';
-import HomeScene from '~/scenes/Home/Home';
-import useOpenSettingsModal from '~/scenes/Modals/useOpenSettingsModal';
-import { PreloadQueryArgs } from '~/types/PageComponentPreloadQuery';
-
-const homeQueryNode = graphql`
-  query homeQuery(
-    $interactionsFirst: Int!
-    $interactionsAfter: String
-    $globalLast: Int!
-    $globalBefore: String
-    $viewerLast: Int!
-    $viewerBefore: String
-    $visibleTokensPerFeedEvent: Int!
-  ) {
-    viewer {
-      ... on Viewer {
-        user {
-          dbid
-        }
-      }
-    }
-
-    ...HomeFragment
-    ...FeedNavbarFragment
-    ...useOpenSettingsModalFragment
-  }
-`;
+import GalleryRedirect from '~/scenes/_Router/GalleryRedirect';
 
 export default function Home() {
-  const query = useLazyLoadQuery<homeQuery>(homeQueryNode, {
-    interactionsFirst: NOTES_PER_PAGE,
-    globalLast: ITEMS_PER_PAGE,
-    viewerLast: ITEMS_PER_PAGE,
-    visibleTokensPerFeedEvent: MAX_PIECES_DISPLAYED_PER_FEED_EVENT,
-  });
-
-  const { viewer } = query;
-  const viewerUserId = viewer?.user?.dbid ?? '';
-  const defaultFeedMode = viewerUserId ? 'FOLLOWING' : 'WORLDWIDE';
-
-  const [feedMode, setFeedMode] = usePersistedState<FeedMode>(FEED_MODE_KEY, defaultFeedMode);
-
-  // This effect handles adding and removing the Feed controls on the navbar when mounting this component, and signing in+out while on the Feed page.
-  useEffect(() => {
-    if (!viewerUserId) {
-      setFeedMode('WORLDWIDE');
-    }
-  }, [viewerUserId, feedMode, setFeedMode]);
-
-  useOpenSettingsModal(query);
-
-  return (
-    <GalleryRoute
-      navbar={<FeedNavbar feedMode={feedMode} onChange={setFeedMode} queryRef={query} />}
-      element={<HomeScene setFeedMode={setFeedMode} feedMode={feedMode} queryRef={query} />}
-    />
-  );
+  return <GalleryRedirect to={{ pathname: '/activity' }} />;
```

Comment:
I kept the /home route around to redirect to the name home page, `/activity`, in case someone has gallery.so/home bookmarked.